src/OversetSimulation.cpp

```diff
@@ -103,8 +103,12 @@ void OversetSimulation::exchange_solution()
     if (m_has_amr) {
         m_tg.dataUpdate_AMR();
     } else {
-        throw std::runtime_error("Invalid overset exchange");
+        const int row_major = 0;
```

Comment:
do we want `else if nalu-wind only and the else original error?

Reply:
I think we have an assumption and a throw elsewhere that guarantees at least one solver is unstructured (nalu-wind)